ntbs-integration-tests/ClinicalDetailsPageTests.cs

```diff
@@ -0,0 +1,173 @@
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using AngleSharp.Html.Dom;
+using ntbs_integration_tests.Helpers;
+using ntbs_service;
+using ntbs_service.Models.Validations;
+using Xunit;
+
+namespace ntbs_integration_tests
+{
+    public class ClinicalDetailsPageTests : TestRunnerBase
+    {
+        protected override string PageRoute
+        {
+            get { return Routes.ClinicalDetails; }
+        }
+        
+        public ClinicalDetailsPageTests(NtbsWebApplicationFactory<Startup> factory) : base(factory) {}
+
+        [Fact]
+        public async Task Post_ReturnsPageWithAllModelErrors_IfModelNotValid()

```

Comment:
I've decided to put in all the scenarios that cause a model error into the same test, rather than doing a different one for each, as this will make running them faster. However, there is obviously merit in having a specific test for each scenario - but it means writing and running a lot more tests. What do you think?

Reply:
This seems like a good compromise - we can expand on it if bugs that this doesn't catch surface up :)